src/ceph-run

```diff
@@ -1,12 +1,18 @@
 #!/bin/sh
 
 sleep=5
+no_restart=0
+
+if [ $1 == "--no-restart" ]; then
+    no_restart=1
+    shift
+fi
 
 while [ true ]; do
     "$@"
     exit_code=$?
     
-    if [ $exit_code -gt 128 ]; then
+    if [ $exit_code -gt 128 ] && [$no_restart -eq 0 ]; then
```

Comment:
Bit of bash nonsense:

```
if [ $exit_code -gt 128 ] && [$no_restart -eq 0 ]; then
```

needs to be

```
if [ $exit_code -gt 128 ] && [ $no_restart -eq 0 ]; then
```

There needs to be a space between [ and $no_restart].